src/components/CommandBar/CommandBarBase/CommandBarBase.module.scss

```diff
@@ -99,4 +100,5 @@ $min-height: calc(9 * var(--spacing-mega));
       animation: overlayHide var(--transition-fast) ease;
     }
   }
+  z-index: var(--z-index-toast);
```

Comment:
this should be called something else. like z-index-modal-overlay or z-index-overlay